### ISSUE TITLE
Vietnamese stacked diacritics

### ISSUE THREAD

**Issue by vv-monsalve** [marcologous/hanken-grotesk#63]
You may want to give more space between diacritics and even to the base letter in some stacked diacritics to improve its legibility. Particularly: Abrevetilde, Acircumflextilde, ahookabove
![Screen Shot 2020-11-10 at 16 06 09](https://user-images.githubusercontent.com/48698976/98746655-ea171000-2383-11eb-8b24-9982af0d19bc.png)

**Comment by marcologous** [comment]
True, it can get really tight when viewed in smaller sizes. I will work on those.

**Comment by vv-monsalve** [comment]
New Vietnamese preview images for Windows 10 Chrome 71 at 16pt.

Thin weight seems pretty light in Chrome.

**Thin, ExtraLight, Light, Regular**
![Desktop_Windows_10_chrome_71 0_](https://user-images.githubusercontent.com/48698976/108291070-a50f3080-715f-11eb-8422-77eb5bd04d10.png)

**Medium, SemiBold, Bold, ExtraBold**
![Desktop_Windows_10_chrome_71 0_](https://user-images.githubusercontent.com/48698976/108291088-ab051180-715f-11eb-8603-c54264a021c1.png)

**Black**
![Desktop_Windows_10_chrome_71 0_](https://user-images.githubusercontent.com/48698976/108291139-bce6b480-715f-11eb-88c9-4562fd68c01f.png)

**Comment by vv-monsalve** [comment]
Hi @marcologous, I'm closing this issue since it was completed a long time ago :)
